src/runtime/components/forms/Checkbox.vue

```diff
@@ -57,7 +57,7 @@ export default defineComponent({
       default: null
     },
     modelValue: {
-      type: [Boolean, Array],
+      type: [Boolean, Array] as PropType<boolean | Array<any> | null>,
```

Comment:
@romhml @benjamincanac In relation to [my message](https://github.com/nuxt/ui/pull/2309#issuecomment-2515569490), shouldn't `null` be included in the type array too?

```ts
{
  type: [Boolean, Array, null] as PropType<boolean | Array<any> | null>
}
```